### PR TITLE
Named functions

### PR DESCRIPTION
**Pull request by ghost**
Created named functions to replace anonymous ones

### CODE REVIEW

**Review by overlordbleck — APPROVED** [review]
Looks good to me 👍

File: lua/cfcfactions/core/cl/cl_clientstartup.lua [hunk 1]
```diff
@@ -44,27 +44,35 @@ function cfcFactions:DisplayMenu()
...
 --If the defined key is properly set, users can use that specific key to also open/close the derma
-hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", function(ply, button)
+local function menuKeyDown(ply, button)
     if input.GetKeyName(button) == cfcFactions.Config.CLIENT_KEY then
         cfcFactions:DisplayMenu()
     end
-end)
+end
+
+hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", menuKeyDown)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/cl_clientstartup.lua]: This hook name should be CamelCase

```suggestion
hook.Add("PlayerButtonDown", "CFC_Fac_MenuKeyDown", menuKeyDown)
```

File: lua/cfcfactions/core/cl/cl_clientstartup.lua [hunk 2]
```diff
@@ -44,27 +44,35 @@ function cfcFactions:DisplayMenu()
...
 
 --If a user requests to display the derma from serverside
-net.Receive('CFC_Fac_ToggleDerma', function(len, ply)
+local function toogleDerma(len, ply)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/cl_clientstartup.lua]: These method names seem a little to vague for me. Best practice is usually to scope them by name in some way.


```suggestion
local function toggleFactionsDerma(len, ply)
```
(You'll need to change the other reference to it, of course)

File: lua/cfcfactions/core/cl/cl_clientstartup.lua [hunk 3]
```diff
@@ -44,27 +44,35 @@ function cfcFactions:DisplayMenu()
...
     cfcFactions:DisplayMenu()
-end)
+end
+
+net.Receive('CFC_Fac_ToggleDerma', toogleDerma)
 
-net.Receive('CFC_Fac_SendMessage', function(len, ply)
+local function sendMessage(len, ply)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/cl_clientstartup.lua]:
```suggestion
local function sendFactionsMessage(len, ply)
```

Can you rename the rest of them in this manner, too?

File: lua/cfcfactions/core/cl/dermas/cl_logsderma.lua [hunk 4]
```diff
@@ -19,18 +19,27 @@ end
 vgui.Register('D_logsdermasderma', Panel)
 
 --TODO: tie in to the system
-net.Receive("FetchAlerts", function()
 
-end)
+local function fetchAlerts()
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_logsderma.lua]:
```suggestion
local function fetchFactionAlerts()
```

and so on

**Review by sarahsturgeon — CHANGES_REQUESTED** [review]

File: lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua [hunk 5]
```diff
@@ -182,18 +182,27 @@ vgui.Register('D_cfcfactionsderma', Panel)
...
-end)
+net.Receive("FactionCreated", factionCreated)
 
-net.Receive("FactionFetchQuery", function()
+local function factionEdited()
+    
+end
 
-end)
+net.Receive("FactionEdited", factionEdited)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_factionsderma.lua]: Can you change these net strings to follow our hook pattern? `CFC_Fac_FactionEdited`

File: lua/cfcfactions/core/cl/dermas/cl_logsderma.lua [hunk 6]
```diff
@@ -19,18 +19,27 @@ end
...
-net.Receive("AddAlert", function()
+net.Receive("FetchAlerts", fetchFactionAlerts)
 
-end)
+local function addFactionAlert()
+    
+end
 
-net.Receive("FetchAdminAlerts", function()
+net.Receive("AddAlert", addFactionAlert)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_logsderma.lua]: Same thing here. `CFC_Fac_AddAlert`

File: lua/cfcfactions/core/cl/dermas/cl_usersderma.lua [hunk 7]
```diff
@@ -20,26 +20,39 @@ end
...
+
+net.Receive("UserUpdateStats", factionUserUpdateStats)
 
-net.Receive("UserDeleted", function()
+local function factionUserDeleted()
+    
+end
 
-end)
+net.Receive("UserDeleted", factionUserDeleted)
```
Comment (sarahsturgeon) [inline lua/cfcfactions/core/cl/dermas/cl_usersderma.lua]: Same thing here.

`CFC_Fac_UserDeleted`

**Review by sarahsturgeon — APPROVED** [review]

**Review by sarahsturgeon — CHANGES_REQUESTED** [review]
Accidentally approved

**Review by sarahsturgeon — APPROVED** [review]
👍 Good job, thank you!

**Pull request merged**